File: MiniDOMWebCtrls/DMD/POP3/EMailFoldersHandler.cs

namespace minidom.Forms
{
    public class EMailFoldersHandler : CBaseModuleHandler
    {
        public EMailFoldersHandler()
        {
        }

        public override Databases.DBObjectCursorBase CreateCursor()
        {
            return new Office.MailFolderCursor();
        }
    }
}

File: MiniDOMWebCtrls/DMD/POP3/EMailFoldersHandler.vb
Imports minidom
Imports minidom.Sistema
Imports minidom.Forms
Imports minidom.Office
Imports minidom.WebSite
Imports minidom.Anagrafica

Imports minidom.Databases
Imports minidom.Net

Namespace Forms



    Public Class EMailFoldersHandler
        Inherits CBaseModuleHandler

        Public Sub New()
            
        End Sub

        Public Overrides Function CreateCursor() As DBObjectCursorBase
            Return New minidom.Office.MailFolderCursor
        End Function


    End Class



End Namespace